mkt/settings.py

```diff
@@ -180,7 +180,6 @@
     'django.contrib.messages.middleware.MessageMiddleware',
     'mkt.api.middleware.APIBaseMiddleware',
     'mkt.api.middleware.AuthenticationMiddleware',
-    'commonware.log.ThreadRequestMiddleware',
```

Comment:
This middleware does 2 things:

- Sets the username of the user from the current request in thread local
- Sets the remote addr from the current request in thread local

It also exposes 2 functions to play with that data, `get_remote_addr()` and `get_username()`. We don't use the latter, and the former was only useful because we were trying to access the remote addr in models. I moved the call to the login view, which was the only place where the model method was used, and as a result we're free to simply use `request.META.get('REMOTE_ADDR')` and we don't need the methods or the middleware anymore.